angularjs-portal-home/src/main/webapp/my-app/marketplace/controllers.js

```diff
@@ -167,6 +174,7 @@ define(['angular', 'jquery'], function(angular, $) {
               //init variables
               $scope.portlets = [];
               marketplaceService.getPortlets().then(function(data) {
+				  console.log(data.portlets);
```

Comment:
Lol oops! That was just there from when I was taking a look at the portlet object. I'll take it out ASAP. Nice catch.

Zeke

Sent from my T-Mobile 4G LTE Device


-------- Original message --------
From: Tim Levett <notifications@github.com>
Date: 7/21/16 4:05 PM (GMT-06:00)
To: UW-Madison-DoIT/angularjs-portal <angularjs-portal@noreply.github.com>
Cc: David Witter <witterdez@hotmail.com>, Author <author@noreply.github.com>
Subject: Re: [UW-Madison-DoIT/angularjs-portal] Materialize marketplace browse view (#485)


In angularjs-portal-home/src/main/webapp/my-app/marketplace/controllers.js<https://github.com/UW-Madison-DoIT/angularjs-portal/pull/485#discussion_r71786923>:

> @@ -167,6 +174,7 @@ define(['angular', 'jquery'], function(angular, $) {
>                //init variables
>                $scope.portlets = [];
>                marketplaceService.getPortlets().then(function(data) {
> +                               console.log(data.portlets);


That would be a lotta console output, may want to only do that if the APP_FLAGS.debug is true.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/UW-Madison-DoIT/angularjs-portal/pull/485/files/9c2d4a66cfae160e7a525d56da3a25d512676fd4#r71786923>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFjJTsmrn1aVf_qtfer7eukTZqE1w1Msks5qX98HgaJpZM4JSNeV>.